eras/shelley-ma/formal-spec/mary-ledger.tex

```diff
@@ -382,7 +382,7 @@
 \bibliography{references}
 
 \begin{appendix}
-  \include{valmonoid-examples}
+  % \include{valmonoid-examples}
```

Comment:
Better to remove it, no?  Don't see any value in keeping a broken reference even commented